### PR TITLE
Rzr-10_footer

### PR DESCRIPTION
**Pull request by SashaGraves**
Footer:
added and styled.

Button template:
added and styled,
+ variables with template classNames for buttons

Header:
link styles a bit changed;
button and text positioned better

Div-container:
added, issues fixed

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- <meta name="viewport" content="width=device-width, initial-scale=1.0"> -->
```
Comment (cashidze) [inline index.html]: We need this line.

File: source/footer/footer-desclaim/footer-desclaim.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.footer-desclaim {
```
Comment (cashidze) [inline source/footer/footer-desclaim/footer-desclaim.scss]: disclaimer

File: source/footer/footer.tsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import FooterDesclaim from './footer-desclaim/footer-desclaim';
```
Comment (cashidze) [inline source/footer/footer.tsx]: Disclaimer

File: source/footer/footer.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+import NavFooter from './nav-footer/nav-footer';
+
+export default () => (
+  <div className="footer">
+    <NavFooter />
+    <FooterDesclaim />
```
Comment (cashidze) [inline source/footer/footer.tsx]: Disclaimer

File: source/footer/nav-footer/nav-footer.scss [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  > .container {
+    box-sizing: border-box;
+    padding-top: 30px;
+    height: 157px;
+
+
+    .contact-block {
+      display: float;
+      float: right;
+
```
Comment (cashidze) [inline source/footer/nav-footer/nav-footer.scss]: Too much empty lines. Also, not sure if tab size is correct or this document

File: source/footer/nav-footer/nav-footer.tsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+import './nav-footer.scss';
+import PhoneNumber from '../../shared/phone-number/phone-number';
+import Button from '../../shared/button/button';
+
+export default () => (
+  <div className="nav-footer">
+    <div className="container">
+      <div className="contact-block">
+        <PhoneNumber />
+        <Button buttonText="Заказать звонок!" clickHandler="" />
```
Comment (cashidze) [inline source/footer/nav-footer/nav-footer.tsx]: buttonText can be replaced with "text" props, still meaningful. Maybe use "label" props name

File: source/header/nav-header/nav-header.scss [hunk 7]
```diff
@@ -38,6 +45,10 @@
   }
 }
 
-.nav-header[class~="light-theme"] {
-  background-color: #949597;
+
+
```
Comment (cashidze) [inline source/header/nav-header/nav-header.scss]: double empty line?

**Review by cashidze — COMMENTED** [review]
Minor comments

File: source/footer/nav-footer/nav-footer.scss (hunk 5 above)
Comment (SashaGraves) [reply source/footer/nav-footer/nav-footer.scss]: ok, lines gone.
tabs are for SCSS nested selectors, they are fine

File: source/header/nav-header/nav-header.scss (hunk 7 above)
Comment (SashaGraves) [reply source/header/nav-header/nav-header.scss]: I LOVE EMPTY LINES

File: source/footer/nav-footer/nav-footer.tsx (hunk 6 above)
Comment (SashaGraves) [reply source/footer/nav-footer/nav-footer.tsx]: I LOVE LONG WORDS but okey, label

File: source/header/phone-header/phone-header.scss [hunk 8]
```diff
@@ -2,34 +2,38 @@
   background: #000000;
   text-align: right;
   font-size: 14px;
-  line-height: 16px;
+  line-height: 23px;
+
```
Comment (cashidze) [inline source/header/phone-header/phone-header.scss:6]: Empty line

File: source/shared/button/button.scss [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+  font-size: 14px;
+  line-height: 16px;
+  color: #FFFFFF;
+  background: transparent;
+  border: 1px solid #FFFFFF;
+  box-sizing: border-box;
+  border-radius: 17px;
+  padding: 8px 30px 9px;
+
+
```
Comment (cashidze) [inline source/shared/button/button.scss]: Empty line

**Review by cashidze — APPROVED** [review]

**Pull request merged**